lib/license.js

```diff
@@ -2,20 +2,24 @@
 
 var _ = require('lodash');
 var assert = require('assert');
+var copyright = require('./copyright');
 var debug = require('debug')('strong-tools:fix-license');
 var exec = require('sync-exec');
 var fs = require('fs');
 var json = require('json-file-plus');
 var path = require('path');
 
-var STRONGLOOP = fs.readFileSync(require.resolve('./STRONGLOOP.tpl'));
-var MIT = fs.readFileSync(require.resolve('./MIT.tpl'));
-var DUAL_MIT = fs.readFileSync(require.resolve('./DUAL_MIT.tpl'));
-var DUAL_ARTISTIC = fs.readFileSync(require.resolve('./DUAL_ARTISTIC.tpl'));
-var ARTISTIC; // FIXME(sam) I'm not sure what it should look like
+// default to current year, possibly overridden later
+var years = (new Date()).getFullYear();
 
+var CUSTOM = 'Unlicensed.';
+if (process.env.SLT_LICENSE) {
+  CUSTOM = fs.readFileSync(process.env.SLT_LICENSE);
+}
+var MIT = fs.readFileSync(require.resolve('./MIT.tpl'));
+var ARTISTIC = fs.readFileSync(require.resolve('./ARTISTIC.tpl'));
 
```

Comment:
This particular command (`slt license`) was written in a purely synchronous style and I didn't want to rewrite it just for this. I would definitely like to in the future, though.